02.calendar/cal.rb

```diff
@@ -0,0 +1,219 @@
+require "date"
+require "optparse"
+
+# 数字表記の月：英語表記の月
+$month_name = {"1" => "Jan", "2" => "Feb", "3" => "Mar", "4" => "Apr", "5" => "May", "6" => "Jun", "7" => "Jul", "8" => "Aug", "9" => "Sep", "10" => "Oct", "11" => "Nov", "12" => "Dec"}
```

Comment:
これでも良いと言えば良いんですが、個人的にはこうやって書いた方が視認性が高くて好みです。実際こう書く人も多いと思いますので、こういう風に書く習慣をつけてもらうといいかもです。

```suggestion
$month_name = {
  "1" => "Jan",
  "2" => "Feb",
  "3" => "Mar",
  "4" => "Apr",
  "5" => "May",
  "6" => "Jun",
  "7" => "Jul",
  "8" => "Aug",
  "9" => "Sep",
  "10" => "Oct",
  "11" => "Nov",
  "12" => "Dec"
}
```

Reply:
よく考えたらこちらも`strftime`でできそうですね。もちろんこのままでも大丈夫です。

Reply:
確かに視認性が高いですね。
hashの書き方を修正しました。